### PR TITLE
Misc minor improvements

### PR DESCRIPTION
**Pull request by jacobmontiel**
- Improved/reworded some docstrings for clarity.
- Fixed minor issues in docstrings that resulted in them not being rendered properly in the documentation.
- Hide internal methods in the STAGGER generator.

### CODE REVIEW

File: river/datasets/synth/sea.py [hunk 1]
```diff
@@ -6,11 +6,19 @@
 class SEA(base.SyntheticDataset):
     """SEA synthetic dataset.
 
-    Each observation is composed of 3 features. Only the first two features are relevant. The
+    Implementation of the data stream with abrupt drift described in [^1]. Each observation is composed of 3 features. Only the first two features are relevant. The
```
Comment (smastelini) [inline river/datasets/synth/sea.py:9]: Is the line character limit per line being exceeded here?

File: river/drift/ddm.py [hunk 2]
```diff
@@ -100,10 +110,6 @@ def reset(self):
     def update(self, value):
         """Update the change detector with a single data point.
 
-        After calling this method, to verify if change was detected or if the learner is in the
-        warning zone, one should call the super method detected_change, which returns `True` if
-        concept drift was detected and `False` otherwise.
-
```
Comment (smastelini) [inline river/drift/ddm.py:106]: Nice!

File: river/drift/eddm.py [hunk 3]
```diff
@@ -18,19 +18,28 @@ class EDDM(DriftDetector):
...
+    * if $(p_i' + 2 * s_i')/(p'_{max} + 2 * s'_{max}) < \alpha$ -> Warning zone
+
+    * if $(p_i' + 2 * s_i')/(p'_{max} + 2 * s'_{max}) < \beta$ -> Change detected
 
     $\alpha$ and $\beta$ are set to 0.95 and 0.9, respectively.
 
+    **Input:** EDDM expects `value` to be a binary value as input, indicating
+     whether the last sample analyzed was correctly classified.
+
+    - 0: Correclty classifies
```
Comment (smastelini) [inline river/drift/eddm.py:38]: There is a typo in "Correctly"

File: river/drift/hddm_a.py [hunk 4]
```diff
@@ -6,11 +6,19 @@
...
+    HDDM_A is a drift detection method based on the Hoeffding’s inequality.
+    HDDM_A uses the average as estimator. It receives as input a stream of real
+    values and returns the estimated status of the stream: STABLE, WARNING or
+    DRIFT.
 
-    Implementation based on MOA.
+    **Input:** HDDM_A expects `value` to be a binary value as input, indicating
+     whether the last sample analyzed was correctly classified.
+
+    - 0: Correclty classifies
```
Comment (smastelini) [inline river/drift/hddm_a.py:17]: Typo in "Correctly"

File: river/drift/hddm_w.py [hunk 5]
```diff
@@ -6,11 +6,19 @@
...
+    HDDM_W is an online drift detection method based on McDiarmid's bounds.
+    HDDM_W uses the Exponentially Weighted Moving Average (EWMA) statistic as
+    estimator. It receives as input a stream of real predictions and returns
+    the estimated status of the stream: STABLE, WARNING or DRIFT.
 
-    Implementation based on MOA.
+    **Input:** HDDM_W expects `value` to be a binary value as input, indicating
+     whether the last sample analyzed was correctly classified.
+
+    - 0: Correclty classifies
```
Comment (smastelini) [inline river/drift/hddm_w.py:17]: Same typo here :D

**Review by smastelini — CHANGES_REQUESTED** [review]
Thanks for the fixes, @jacobmontiel! They are more than welcome. I left some minor comments regarding typos and line breaks.

File: river/datasets/synth/sea.py (hunk 1 above)
Comment (jacobmontiel) [reply river/datasets/synth/sea.py]: Nice catch. Fixed in 27ef120bd2b0d3c4e1560e223db91ffba0415b16

File: river/drift/eddm.py (hunk 3 above)
Comment (jacobmontiel) [reply river/drift/eddm.py]: Reworded this comment for clarity in 27ef120bd2b0d3c4e1560e223db91ffba0415b16

Same applies to `DDM`, `EDDM`, `HDDM_A`, and `HDDM_W`.

File: river/drift/hddm_a.py (hunk 4 above)
Comment (jacobmontiel) [reply river/drift/hddm_a.py]: Reworded this comment for clarity in 27ef120bd2b0d3c4e1560e223db91ffba0415b16

Same applies to `DDM`, `EDDM`, `HDDM_A`, and `HDDM_W`.

File: river/drift/hddm_w.py (hunk 5 above)
Comment (jacobmontiel) [reply river/drift/hddm_w.py]: Reworded this comment for clarity in 27ef120bd2b0d3c4e1560e223db91ffba0415b16

Same applies to `DDM`, `EDDM`, `HDDM_A`, and `HDDM_W`.

**Comment by smastelini** [comment]
Thanks for the last changes, @jacobmontiel! The PR is ready to be merged!

**Review by smastelini — APPROVED** [review]

**Pull request merged**